python/python/bystro/prs/preprocess_for_prs.py

```diff
@@ -0,0 +1,113 @@
+"""Process dosage matrix, GWAS summary statistics, and LD maps for PRS C+T calculation."""
+
+import glob
+import logging
+import re
+import pandas as pd
+from pyarrow import feather  # type: ignore
+
+logger = logging.getLogger(__name__)
+
+
+def _load_preprocessed_dosage_matrix(dosage_matrix_path: str) -> pd.DataFrame:
+    """Load in dosage matrix generated from bystro-vcf."""
+    return feather.read_feather(dosage_matrix_path)
```

Comment:
This will not work for any meaningfully large dataset. This has to use the approach seen in the prs tests.

Reply:
//2024-03-12 We talked about this, and agreed that in v1 we go with this approach and next PR we'll switch to the memory efficient one.